tools/run-all-tests.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/env bash
+
+set -euo pipefail
+
+script_dir="$(readlink -f "$(dirname "$0")")"
+stdb_root="$(realpath "$script_dir/../")"
+
+set -euox pipefail
+
+cd "$stdb_root"
+
+tools/clippy.sh
```

Comment:
this requires `tools/clippy.sh` to be executable. I pushed a commit to make both of these executable (so we can run them both directly rather than e.g. `bash tools/clippy.sh`)

Reply:
Thank you. For some reason my git executable kept ignoring when I did that, no idea why.

Reply:
Yeah I was wondering if you were on Windows in some form. I think it comes down to the fact that Windows doesn't have the same filesystem bits for controlling permissions. There's some other way to manually force the changes into git..

Reply:
Looks like this is how to write it directly into the git index:
```bash
git update-index --chmod=+x <file>
```
https://stackoverflow.com/questions/6476513/git-file-permissions-on-windows